src/types/types.ts

```diff
@@ -1108,15 +1108,58 @@ export const ErrorStatusCode = {
 export type ErrorStatusCode =
   (typeof ErrorStatusCode)[keyof typeof ErrorStatusCode];
 
-export type TokenIndexRules = {
-  [field: string]: any;
```

Comment:
This is not valid, looking at the rust source code I couldn't find such a thing.